### PR TITLE
Update dependencies and changelog for version 8.5.0.

### PR DESCRIPTION
**Pull request by tarigo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 8.5.0 - November 8, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v8.5.0-beta.1...android-v8.5.0) since [Mapbox Maps SDK for Android v8.5.0-alpha.4](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v8.5.0-beta.1):
+### Performance improvements
+ - Convert GeoJSON features to tiles for the loaded source description in a background thread and thus unblock the UI thread [#15885](https://github.com/mapbox/mapbox-gl-native/pull/15885)
```
Comment (tobrun) [inline CHANGELOG.md]: I'm not seeing this change on release-sangria upstream. Can you add entries for the items missing from this repository gl-native submodule pin vs the latest of release-sangria in gl-native?

Comment (tarigo) [reply CHANGELOG.md]: Yep, rolled back.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 8.5.0 - November 8, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v8.5.0-beta.1...android-v8.5.0) since [Mapbox Maps SDK for Android v8.5.0-alpha.4](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v8.5.0-beta.1):
+### Bug fixes
+ - Fixed the rendering bug caused by redundant pending requests for already requested images [#15864](https://github.com/mapbox/mapbox-gl-native/pull/15864)
+
```
Comment (tobrun) [inline CHANGELOG.md]: can we add a line for https://github.com/mapbox/mapbox-gl-native/pull/15862 ?

Comment (tarigo) [reply CHANGELOG.md]: Done.

**Pull request merged**